tyk-headless/values.yaml

```diff
@@ -198,3 +198,10 @@ pump:
   mounts: []
 
 rbac: true
+
+persistence:
```

Comment:
I would recommend putting this value under `gateway` since it is a pvc of `gateway`. Meanwhile renaming `templates/pvc.yaml` into `templates/pvc-gateway.yaml` to follow the previous convention of this repo and avoid confusion.